### DISCUSSION TITLE
How to use variables in yaml file?

### DISCUSSION THREAD

**Discussion by sarthaks20** [peridigm/peridigm#342] [Q&A]
I followed the Peridigm user guide to use variables in the yaml file.
![image](https://github.com/peridigm/peridigm/assets/72159252/b9d25e70-4dfa-4f57-8028-e14041b319a2)

My code:

![image](https://github.com/peridigm/peridigm/assets/72159252/479c4727-0b24-484c-9240-34b9786fe03b)

Error I am getting:
![image](https://github.com/peridigm/peridigm/assets/72159252/5845f354-14f9-45d9-9cae-c6b471d7a293)

Can anyone please help?

Thanks

**Comment by nsy19971013** [comment]
horizon Must be a specific value, recommended to be 3.015 times the grid size。like horizon: 3.0
